### DISCUSSION TITLE
Error dalam code untuk pembuatan plot ini

### DISCUSSION THREAD

**Discussion by mosesnugraha** [teamalgoritma/community#555] [Q&A] [answered]
apakah ada yang salah dalam code ini? Saya terasa sudan benar, dan berulang kali saya peritas penalizan codenya namun saya running berkali2 tetap aja error.

<img width="1440" alt="Screen Shot 2023-02-21 at 11 11 28" src="https://user-images.githubusercontent.com/123171973/220245946-9db7fea6-fab7-4b66-8909-222bf3cd1d69.png">

**Comment by divakartika** [comment, marked as answer]
Dear @mosesnugraha ,

Dari error nya tertulis "object 'complaint' not found", dapat dipastikan apakah terdapat kolom `complaint` dalam dataframe `top_30issue`? (Dapat dicek dengan `str(top_30issue)` atau `glimpse(top_30issue)`).

Best Regards,
Team Algoritma
